po/hi.po

```diff
@@ -0,0 +1,117 @@
+"Last-Translator: Scrambled777 <weblate.scrambled777@simplelogin.com>\n"
+"Language-Team: Hindi\n"
+"Language: hi\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Gtranslator 46.0\n"
+
+#: data/io.github.zefr0x.hashes.desktop.in:5
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:6 hashes/ui/main_window.py:29
+msgid "Hashes"
+msgstr "Hashes"
+
+#: data/io.github.zefr0x.hashes.desktop.in:6
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:7
+msgid "Identify hashing algorithms"
+msgstr "हैशिंग कलन विधि को पहचानें"
+
+#: data/io.github.zefr0x.hashes.desktop.in:7
+msgid "Hash Identifier"
+msgstr "हैश पहचानकर्ता"
+
+#: data/io.github.zefr0x.hashes.desktop.in:13
+msgid "hash;hashcat;john;detect;identify"
+msgstr "hash;hashcat;john;detect;identify;हैश;हैशकैट;जॉन;पता लगाएं;पहचाने;"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:9
+msgid ""
+"This utility lets you conveniently find out what hashing algorithm has been "
+"used on a particular string."
+msgstr ""
+"यह उपयोगिता आपको आसानी से यह पता लगाने देती है कि किसी विशेष स्ट्रिंग पर किस हैशिंग "
+"कलन विधि का उपयोग किया गया है।"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:10
+msgid "Features:"
+msgstr "विशेषताएँ:"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:12
+msgid "📺 Popularity Ratings - Most popular hashes showed first."
+msgstr "📺 लोकप्रियता रेटिंग - सबसे लोकप्रिय हैश सबसे पहले दिखाए गए।"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:13
+msgid ""
+"👵 Updated! - A new hash algorithm database is used in the identification "
+"process."
+msgstr ""
+"👵 अद्यतन! - पहचान प्रक्रिया में एक नए हैश कलन विधि डेटाबेस का उपयोग किया जाता है।"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:17
+msgid "zefr0x"
+msgstr "zefr0x"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:26
+msgid "hash"
+msgstr "हैश"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:27
+msgid "hashcat"
+msgstr "हैशकैट"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:28
+msgid "john"
+msgstr "जॉन"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:29
+msgid "detect"
+msgstr "पता लगाएं"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:30
+msgid "identify"
+msgstr "पहचाने"
+
+#: hashes/ui/main_window.py:48
+msgid "Enter a hash text"
+msgstr "एक हैश पाठ दर्ज करें"
+
+#: hashes/ui/main_window.py:63
+msgid "Waiting for input…"
+msgstr "इनपुट की प्रतीक्षा में…"
+
+#: hashes/ui/main_window.py:65
+msgid "Can't find an algorithm that matches your input!"
+msgstr "आपके इनपुट से मेल खाने वाला कलन विधि नहीं मिल सकी!"
+
+#: hashes/ui/main_window.py:154
+msgid "<b>Extended!</b>"
+msgstr "<b>विस्तारित!</b>"
+
+#: hashes/ui/main_window.py:165
+msgid "Hashcat"
+msgstr "हैशकैट"
+
+#: hashes/ui/main_window.py:179
+msgid "John"
+msgstr "जॉन"
+
+#: hashes/ui/main_window.py:208
+msgid "translator-credits"
+msgstr "Scrambled777 <weblate.scrambled777@simplelogin.com>, 2024"
```

Comment:
```suggestion
msgstr "Scrambled777 <weblate.scrambled777@simplelogin.com>"
```

You shouldn't add the year here. The UI is not designed to handle multiple translators (it will just be ignored).